src/api/Instance/instance.ts

```diff
@@ -1,7 +1,8 @@
 import axios, { AxiosHeaders, AxiosRequestConfig } from 'axios';
 
+export const API_URL = 'http://localhost:5000/api';
```

Comment:
can we take it from .env file? or at least set in package json script?

Reply:
i created .env.local.file